guides/infra/bedrock-infra-rollback.md

```diff
@@ -0,0 +1,221 @@
+# Terraform Rollback Implementation using Bedrock CLI
+
+| Revision | Date         | Author         | Remarks       |
+| -------: | ------------ | -------------- | ------------- |
+|      0.1 | Apr-20, 2020 | Nathaniel Rose | Initial Draft |
+|      0.1 | May-11, 2020 | Nathaniel Rose | Updated Draft |
+
+## 1. Overview
+
+It is highly desireable to have a fully-automated Continuous Delivery pipeline
+in place to assist with testing and validation of source code changes made to
+the application services and supporting infrastructure. Without such a system,
+it is difficult to have any degree of confidence that a new deployment will
+succeed without introducing new regressions. However, high velocity of changes
+with distributed teams may eventually lead to the scenario in which a production
+environment transitions into a failed state due to an improperly tested feature
+added.
+
+This document outlines multiple solutions and approaches that seeks to provide a
```

Comment:
should it be "that seek to provide a rollback mechanism" or "that seek to provide rollback mechanisms" instead of "that seeks"?

Reply:
Updated.